src/policy/policy.cpp

```diff
@@ -266,13 +273,18 @@ bool IsWitnessStandard(const CTransaction& tx, const CCoinsViewCache& mapInputs)
 
         // Check policy limits for Taproot spends:
         // - MAX_STANDARD_TAPSCRIPT_STACK_ITEM_SIZE limit for stack item size
-        // - No annexes
+        // - Limited annex format
```

Comment:
If the 0x50 <1 byte data len> <len bytes> start to be used by more experimental applications on inquisition, of course we might have conflicts with assigning consensus semantic for taproot annex, maybe annexcarrier could be its own annex "tag" by introducing a marker value before the len for a v0.2 of annexcarrier. I definitely can see a "annexcarrier" generic data payload where there is no meaning attached to the record itself, for client-reserved value (like nSequence was used sometimes in the past)

Reply:
Per bitcoin/bips#1381 `0x50 n [n-bytes]` (where `n` is a single byte equal or less than `0x7e`) is interpreted as an annex entry with type 0 and contents of `n-bytes`, so this is effectively just reserving type 0 for data that doesn't have any additional consensus meaning (though may be accessible via a future `OP_PUSHANNEX`). Per that spec, to extend it to other types (which may have additional consensus meaning), you'd use a multibyte `n` where the first byte has the high bit set, and to extend to longer than 126 bytes you'd say `0x50 0x7F [n-0x7F] [n-bytes]`.